src/backend/aspen/api/main.py

```diff
@@ -107,32 +107,11 @@ def get_app() -> FastAPI:
         prefix="/v2/lineages",
         dependencies=[Depends(get_auth_user)],
     )
-    _app.include_router(
-        phylo_runs.router,
-        prefix="/v2/phylo_runs",
-        dependencies=[Depends(get_auth_user)],
-    )
-    _app.include_router(
-        phylo_trees.router,
-        prefix="/v2/phylo_trees",
-        dependencies=[Depends(get_auth_user)],
-    )
-    _app.include_router(
-        samples.router,
-        prefix="/v2/samples",
-        dependencies=[Depends(get_auth_user)],
-    )
```

Comment:
I refactored these into the loop below -- we're removing these routes once the FE is fully converted.